diff --git a/python/uv/BUILD.bazel b/python/uv/BUILD.bazel
index 7ce6ce05..6fc5b8a4 100644
--- a/python/uv/BUILD.bazel
+++ b/python/uv/BUILD.bazel
@@ -37,11 +37,7 @@ current_toolchain(
     # Marked manual so that `bazel test //...` passes
     # even if no toolchain is registered.
     tags = ["manual"],
-    # EXPERIMENTAL: Visibility is restricted to allow for changes.
-    visibility = [
-        "//:__subpackages__",
-        "@rules_python//examples:__subpackages__",
-    ],
+    visibility = ["//visibility:public"],
 )
 
 bzl_library(
